libcasr/src/report.rs

```diff
@@ -479,7 +479,30 @@ impl CrashReport {
             return None;
         }
 
-        if let Ok(file) = std::fs::File::open(&debug.file) {
+        // Search for file
```

Comment:
Move to casr-java. Just modify DebugInfo.file

Reply:
done